### DISCUSSION TITLE
Cloning data from RECAP using clone_from_cl

### DISCUSSION THREAD

**Discussion by v-anne** [freelawproject/courtlistener#4528] [Q&A] [answered]
I'm attempting to use `clone_from_cl.py` to clone data from the RECAP API into my local environment.

I am getting the following error: `Error: CL_API_TOKEN not set in .env file`

I am using the .env.dev file, as instructed on the developer guide, and I added a variable as follows to that file:
`CL_API_TOKEN="{my API key}"`

`DEVELOPMENT=on`

Am I not supposed to use an .env.dev file?

**Comment by ERosendo** [comment, marked as answer]
@v-anne did you try recreating your containers after updating the `.env.dev` file?

Here's a quick way to do it:

```
docker compose up -d 
```

**Reply by v-anne** [reply]
Thanks, @ERosendo. Is there any chance I could get permission to clone docket entries using the API? This is for my work on #4519.

**Reply by mlissner** [reply]
I can set you up. Email me your username: mike@free.law. Thank you!
